### DISCUSSION TITLE
TikTok Full Size DP addition

### DISCUSSION THREAD

**Discussion by viczav** [Ademking/fb-profile-picture-viewer#11] [Ideas]
I've recently noticed _TikTok_ (web version) has limited the size of DP to _100x100_ if you open it in a new tab. So, my question is if there's a chance to add TikTok in the profile-picture-viewer extension? Thanks in advance!
![Screenshot 2022-03-13 112041](https://user-images.githubusercontent.com/15375695/158069066-129dca67-d9ea-4ec3-a150-cb5ec3d0af71.png)

**Comment by Ademking** [comment]
@viczav Thank you so much for your suggestion! I will add this feature soon 👍

**Comment by Ademking** [comment]
@viczav Update:

It's working now! This feature will be available in v1.0.7

![image](https://user-images.githubusercontent.com/12462188/158084606-256b7a02-c7ec-443b-853f-97262b9484e2.png)

**Comment by Ademking** [comment]
@viczav Check the latest update

**Reply by viczav** [reply]
Omg, that was so fast. Thank you so much for that addition, I really appreciate it!
